tests/Metadata/Driver/xml/MultilineGroupsFormat.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="JMS\Serializer\Tests\Fixtures\MultilineGroupsFormat"
+           exclusion-policy="all"
+           read-only="true"
+    >
+        <virtual-property method="getAmount" type="integer" serialized-name="amount">
+            <groups>
+                <value>first.test.group</value>
+            </groups>
+        </virtual-property>
+        <virtual-property method="getCurrency" type="string" serialized-name="currency">
+            <groups>
+                <value>first.test.group</value>
```

Comment:
can you add here one more item, something like `<value>second.test.group</value>`